src/docs/release.md

```diff
@@ -140,9 +140,9 @@ whether a release is needed from a stable branch.
 ---------------------
 
 A dry run is not strictly required since CI includes one, but you might
-like to try one anyway; if so, releases should only be published from
-master, so get on master and ensure your version number commit is present.
-After confirming this, run.
+like to try one anyway. To do so, switch to your release branch (which will either be `master` for
+a unstable weekly release, or a release branch like `1.9.x` for a stable release), and ensure that
```

Comment:
This makes a *lot* more sense now, and I think making the distinction between weekly and stable is important.